CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
 ## 1.9.0
 Release built: _not released yet_
 
+### What’s new?
+- Added support for the `cuttlefish` protocol version.
+
+### API Changes
+- Added a new `/transaction/subintent-status` endpoint to check the status of a transaction subintent.
+- Added two new optional fields to the `/transaction/committed-details` endpoint: `subintent_details` and `child_subintent_hashes`, which provide information about transaction subintents if present.
+- Added a new `/transaction/preview-v2` endpoint to preview transactions. This supports V2 transactions and beyond. If you still need to preview V1 transactions, use the `/transaction/preview` endpoint instead.
+
+### Database changes
+- New `ledger_subintents` table that stores information about subintent status.
+- New `UserV2` ledger transaction type discriminator in the `ledger_transactions` table.
+- New `ledger_transaction_subintent_data` table that stores additional information about the transaction's subintents.
```

Comment:
The `ledger_subintents` and `ledger_transaction_subintent_data` look a little bit inconsistent. Also out of interest, do they both only capture the data of `finalized` subintents?

Perhaps we should call them `ledger_finalized_subintents` and `ledger_finalized_subintent_data`?

Reply:
`ledger_subintents` ->  it captures only finalized. I can rename that to `ledger_finalized_subintents`.

`ledger_transaction_subintent_data` -> right now we store only finalized. but that's maybe a good question if we should. Maybe we should return it for all transactions? even failed ones? 

What other naming do you propose if you feel that it's not consistent?

Reply:
Well I guess for consistency, I'd perhaps expect either: `ledger_subintents` & `ledger_subintent_data` or `ledger_finalized_subintents` & `ledger_finalized_subintent_data`. But it's not a big deal :)

Reply:
`ledger_transaction_subintent_data` is just sub table with 1:1 relation to `ledger_transaction` table, that's why i have left full `ledger_transaction_` prefix, we do have same naming strategy for `ledger_transaction_events` table. I'd prefer to keep it like that. It also offers different perspective as there's only one row per transaction where `ledger_subintents` is subintent oriented and has 1 row per subintent.

Reply:
Sounds good :). Happy to keep it as `ledger_transaction_subintent_data`